bindata/network/multus/multus.yaml

```diff
@@ -0,0 +1,78 @@
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: multus
+  namespace: multus
+  annotations:
+    kubernetes.io/description: |
+      This daemon set launches the Multus networking component on each node.
+spec:
+  selector:
+    matchLabels:
+      app: multus
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: multus
+        component: network
+        type: infra
+        openshift.io/component: network
+      annotations:
+        scheduler.alpha.kubernetes.io/critical-pod: ''
+    spec:
+      hostNetwork: true
+      nodeSelector:
+        beta.kubernetes.io/os: linux
+      tolerations:
+      - operator: Exists
+      serviceAccountName: multus
+      initContainers:
+      - name: cni-plugins-supported
+        image: {{.CNIPluginsSupportedImage}}
+        command: ["/bin/sh"]
+        args: ["-c", "cp -rf /usr/src/plugins/bin/* /host/opt/cni/bin"]
+        securityContext:
+          privileged: true
+        volumeMounts:
+        - mountPath: /host/opt/cni/bin
+          name: cnibin
+      - name: cni-plugins-unsupported
+        image: {{.CNIPluginsUnsupportedImage}}
+        command: ["/bin/sh"]
+        args: ["-c", "cp -rf /usr/src/plugins/bin/* /host/opt/cni/bin"]
+        securityContext:
+          privileged: true
+        volumeMounts:
+        - mountPath: /host/opt/cni/bin
+          name: cnibin
+      containers:
+      - name: kube-multus
+        image: {{.MultusImage}}
+        command: ["/entrypoint.sh"]
+        args:
+        - "--multus-conf-file=auto"
+        - "--multus-kubeconfig-file-host=/etc/kubernetes/cni/net.d/multus.d/multus.kubeconfig"
+        - "--namespace-isolation=true"
+        resources:
+          requests:
+            cpu: "100m"
+            memory: "50Mi"
```

Comment:
This says that for scheduling purposes, kube should assume that the pod will use 50M of RSS, and 100 milli-CPUs (aka 10% of a single CPU). The multus pod is basically idle after initial setup, so those numbers are definitely too high, and will prevent the node from being effectively scheduled to capacity.

Probably best to just drop the `resources` section entirely

Reply:
I agree it's probably too high, but resources were recently added to every operator and pod by @sjenning.  Example: https://github.com/openshift/cluster-node-tuning-operator/commit/2f6d6179e649e4c15f25df75277a4756eb573465

@sjenning what values would be consistent with the rest of your PRs from last week?

Reply:
on discussion with casey, we're just removing this section as the container basically idles after startup